hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestHRegion.java

```diff
@@ -7364,4 +7367,154 @@ protected HStoreForTesting(final HRegion region,
       return super.doCompaction(cr, filesToCompact, user, compactionStartTime, newFiles);
     }
   }
+
+  @Test(timeout=20000)
```

Comment:
Usually we do not add this to a single test as we have a test timeout for the whole test file.

Reply:
Ok